scipy/sparse/linalg/isolve/meson.build

```diff
@@ -0,0 +1,95 @@
+fortran_ignore_warnings = [
+  '-Wno-tabs', '-Wno-conversion',
+  '-Wno-argument-mismatch', '-Wno-unused-dummy-argument',
+  '-Wno-maybe-uninitialized', '-Wno-unused-label',
+  '-Wno-unused-variable']
+
+# Note: 3 source files in iterative/ are unused
+methods = ['iterative/BiCGREVCOM.f.src',
+  'iterative/BiCGSTABREVCOM.f.src',
+  'iterative/CGREVCOM.f.src',
+  'iterative/CGSREVCOM.f.src',
+  'iterative/GMRESREVCOM.f.src',
+  'iterative/QMRREVCOM.f.src']
+Util = ['iterative/getbreak.f.src']
+
+methods_f_0 = custom_target('methods_f_0',
+  output : ['BiCGREVCOM.f'],
+  input : [methods[0]],
+  command : [py3, generate_f2pymod, '@INPUT@', '-o', '@OUTDIR@'],
```

Comment:
This is a bit repetitive. I'm not sure `foreach` can be combined with `custom_target` (the name of the object created may clash), but it's worth a try I think. See https://mesonbuild.com/Syntax.html#foreach-with-an-array

Reply:
I used it in `sparse/csgraph/meson.build` - it's quite elegant syntax